composer.json

```diff
@@ -33,7 +33,7 @@
         "monolog/monolog": "^2.0",
         "predis/predis": "1.1.*",
         "psr/log": "^1",
-        "ramsey/uuid": "3.7.* || 3.8.* || ^4",
+        "ramsey/uuid": "3.7.* || 3.8.* || 3.9.* || ^4",
```

Comment:
It's better to add `^3.7 || ^4`. In this case, you won't have to change the composer.json when new minor versions are released.

Reply:
forced fix, thanks